src/server/meta_store.cpp

```diff
@@ -52,38 +52,64 @@ uint64_t meta_store::get_last_manual_compact_finish_time() const
 uint64_t meta_store::get_decree_from_readonly_db(rocksdb::DB *db,
                                                  rocksdb::ColumnFamilyHandle *meta_cf) const
 {
+    std::string str_last_flushed_decree;
     uint64_t last_flushed_decree = 0;
-    auto ec = get_value_from_meta_cf(db, meta_cf, true, LAST_FLUSHED_DECREE, &last_flushed_decree);
+    auto ec = get_string_value_from_meta_cf(
```

Comment:
Why not use `get_value_from_meta_cf` directly?

Reply:
`get_value_from_meta_cf` reads data from `_db`, but `get_string_value_from_meta_cf` reads data from a readonly db.

Reply:
So is `get_value_from_meta_cf` useless？

I see `get_value_from_meta_cf` has been refactor using `get_string_value_from_meta_cf` and the code is almost duplicate with here, maybe should offer  `get_value_from_meta_cf` overload-version supporting pass `db` and `meta_cf`?